puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline {
+    agent { label 'fast' }
+
+    stages {
+        stage('Setup workspace') {
+            steps {
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: 'update_jobs']],
+                    extensions: [[$class: 'CleanCheckout']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/zjhuntin/foreman-infra.git']
+                    ]
+                ])
+
+                withVenv('./ci', 'pip install jenkins-job-builder')
```

Comment:
Maybe `VirtEnv` ? I worry that `Venv` won't be obvious when reading down the road.